ci/replace_string.py

```diff
@@ -12,12 +13,6 @@ def replace_string_in_file(file_path: str, original_str: str, new_str: str):
         if original_str.encode() not in content:
             raise ValueError("Original string not found in the file")
 
-        # Check if there are more than one matches of the original string
-        if content.count(original_str.encode()) > 1:
-            raise ValueError(
-                "More than one match of the original string found in the file"
-            )
-
```

Comment:
Could you explain why do we now expect more than one occurrence of the placeholder?

Reply:
Somehow `macos aarch64` has two occurences of version string, therefore, I removed that exception and replaced 2 places in `macos aarch64`.

![image](https://github.com/NordSecurity/libtelio/assets/69630990/ea1fc861-e6ed-4caa-969a-f695a834f7d6)

Reply:
That is strange, I just built libtelio.a from this branch and it looks like there is only one match:
```
$ rg --binary --count VERSION_PLACEHOLDER target/debug/libtelio.a
1
$ rg --binary --count VERSION_PLACEHOLDER target/release/libtelio.a
1
```

Reply:
what if you were to download from gitlab artifacts?

Reply:
created issue at Jira LLT-5001